include/bbp/sonata/config.h

```diff
@@ -460,13 +460,17 @@ class SONATA_API SimulationConfig
         double ampStart{};
         /// The final current when a stimulus concludes (nA)
         double ampEnd{};
+        /// Whether this input represents a physical electrode. Default is false
+        bool representsPhysicalElectrode = false;
```

Comment:
do all inputs use this? should it go in `InputBase`?

Reply:
No, `seclamp` does not use this (since it seemed like it would be confusing to allow `seclamp` to use anything other than the builtin SEClamp mechanism)

Reply:
ok, sounds good